### PR TITLE
Resolves #100 Create an AWS S3 Get Object connector

### LINKED ISSUE #100: Create an AWS S3 Get Object connector
This connector will be used to download a single file from a specific bucket in S3.

### PR DESCRIPTION
**Pull request by oscerd**

### CODE REVIEW

**Comment by oscerd** [comment]
@zregvart @rhuss this is the get Object connector. Currently I set deleteAfterRead to false, since the default value is true.

File: connectors/aws-s3-get-object-connector/src/main/resources/camel-connector-schema.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    "javaType":"io.syndesis.s3.get.S3GetObjectConnectorComponent",
+    "groupId":"io.syndesis",
+    "artifactId":"aws-s3-get-object-connector",
+    "version":"0.5-SNAPSHOT"
+  },
+  "componentProperties":{
+    
+  },
+  "properties":{
+    "accessKey":{
```
Comment (rhuss) [inline connectors/aws-s3-get-object-connector/src/main/resources/camel-connector-schema.json:23]: I know I asked in another issue (cant find it though ;-), but is accessKey / secretKey authentication the only way to secure AWS ? Or is there also an alternative like OAuth ? (like e.g. it is for the salesforce and twitter connector)

**Review by rhuss — APPROVED** [review]
LGTM, just a question. See inline.

File: connectors/aws-s3-get-object-connector/src/main/resources/camel-connector-schema.json (hunk 1 above)
Comment (oscerd) [reply connectors/aws-s3-get-object-connector/src/main/resources/camel-connector-schema.json:23]: As far as I know it's only way at the moment. 

http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html

**Comment by oscerd** [comment]
I'll start to work on the polling consumer.

**Pull request merged**